src/images/projectIcons/servo.svg

```diff
@@ -1 +1 @@
.142.127.034.208-.556.31-.674.332-.404.075-.856.138-1.198.374.447-.101.89-.286 1.357-.317.306-.02.547-.013.827-.141.291-.133.67-.194.983-.113.186.048.572.2.408.443l-.013.018c-.123.167-.318.234-.53.248a2.08 2.08 0 0 1-.532-.048l-.012-.002-.023-.005a5.293 5.293 0 0 1-.253-.061l.054.049.017.015a1.92 1.92 0 0 0 .096.078l.02.016.043.031.015.011.059.04.01.008.05.03.02.013.041.024.022.012.043.024.02.01.054.028c.003 0 .005.002.008.004l.065.03.015.006.05.02.02.009a1.933 1.933 0 0 0 .132.048 2.3 2.3 0 0 0 .201.056l.02.004c.017.004.033.008.05.01l.018.005.058.01.01.002.07.01h.003l.063.007.016.002.053.004.017.001.054.003h.014c.045.002.091.002.137.001h.007l.059-.003.014-.001a1.92 1.92 0 0 0 .054-.004l.013-.002.059-.006.007-.001c.044-.006.088-.014.132-.022l.009-.002.054-.012.013-.003a2.216 2.216 0 0 0 .201-.062l.01-.003c.05-.02.099-.04.146-.064l.01-.005.07-.036.004-.003.063-.038.008-.005a.993.993 0 0 0 .063-.043l.01-.007.056-.044.005-.003a1.389 1.389 0 0 0 .17-.168l.01-.011.046-.058.006-.009.039-.055.007-.013c.014-.02.026-.042.039-.063l.007-.014.031-.061.005-.01.03-.07.006-.016.027-.075c.038-.12.08-.225.125-.321.255-.536.624-.751 1.068-1.183a5.113 5.113 0 0 0 .821-1.042c.902-1.484-.264-1.431.054-2.12-.212.159-.902.53-2.015 1.325.318-.477 1.22-1.75 1.22-1.75s-.478.266-1.22.637c.053-.265.53-.69 1.325-1.538-.053 0-.742.371-2.014.69.212-.16 1.643-1.06 1.908-1.75.265-.69 1.485-1.273 1.803-2.068-.795.212-1.59.954-2.333 1.59-.742.637-1.644.69-2.65.69.393-.563 1.323-.432 1.84-.883.54-.474.818-1.194.988-1.874-.242.335-.44.657-.634 1.02.135-.488.248-.984.374-1.477.086-.337.174-.739.377-1.03.002.184.09.352.08.53.182-.488.483-.954.697-1.434.127-.283.203-.704.4-.94.077.399.076.764.105 1.152.211-.142.34-.459.49-.665.303-.42.678-.777.92-1.242.18-.345.188-.69.188-1.063-.03.294-.26.56-.396.804-.11.196-.137.416-.242.606.12-.478.178-.993.09-1.485-.072-.404-.186-.94-.015-1.328-.19.357-.279.769-.38 1.105 0-.694-.003-1.293.383-1.89.174-.27.34-.612.537-.852-.005.227.034.542.071.809.049.352-.02.701-.003 1.05.122-.431.44-.774.55-1.22.09-.36.08-.725.14-1.083.138.269.263.699.23 1.006.304-.85.395-1.595.26-2.485a8.78 8.78 0 0 1-.094-1.321c0-.206-.028-.472.09-.64 0 .43.428.802.475 1.198.028-.604.441-1.165.452-1.783-.237-.154-.448-.466-.712-.486-.37-.027-.323.276-.579.359-.264.085-.468-.024-.756-.06-.408-.053-.908.009-1.327.065a.61.61 0 0 0-.059.012c-.179.043-.383.161-.552.155-.312-.012-.529-.273-.8-.373-.412.64-.872.157-1.227-.213-.238.143-.316.609-.572.724-.434.194-.434-.178-.743-.36-.395-.233-.114-.145-.533.11-.272.164-.686.492-.812-.003a.516.516 0 0 1-.081.063l-.014.009-.047.03h-.002l-.053.029-.017.008a2.082 2.082 0 0 1-.042.02l-.018.01a2.414 2.414 0 0 1-.191.08l-.018.007a6.456 6.456 0 0 1-.133.049l-.017.006a11.73 11.73 0 0 1-.053.019l-.014.005a9.943 9.943 0 0 1-.05.017c-.097.033-.188.064-.26.09l-.022.008c-.075.025-.154.04-.234.05-.273.031-.565-.004-.792.091-.28.118-.356.53-.694.63-.135.04-.333-.127-.473-.054-.403.207-.321.8-.69 1.073-.45-.267-.277-.552-.897-.33-.26.094-.638.266-.867.422
```

Comment:
<img width="121" alt="Screen Shot 2020-09-30 at 9 19 03 AM" src="https://user-images.githubusercontent.com/3766511/94690965-bc5b9780-02fe-11eb-8b2c-1219a01ab58a.png">

<img width="78" alt="Screen Shot 2020-09-30 at 9 19 07 AM" src="https://user-images.githubusercontent.com/3766511/94690964-bc5b9780-02fe-11eb-9de1-18ad35ec6974.png">

Same here. Probably best to just use the original.

Reply:
Ok!